### PR TITLE
Update API.rst

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

File: spikeinterface/core/channelslicerecording.py [hunk 1]
```diff
@@ -8,6 +8,9 @@
 class ChannelSliceRecording(BaseRecording):
     """
     Class to slice a Recording object based on channel_ids.
+
+    Do not use this classe directly but use `recording.channel_slice(...)`
```
Comment (alejoe91) [inline spikeinterface/core/channelslicerecording.py:12]:
```suggestion
    Do not use this class directly but use `recording.channel_slice(...)`
```

File: spikeinterface/core/frameslicerecording.py [hunk 2]
```diff
@@ -6,6 +6,9 @@ class FrameSliceRecording(BaseRecording):
     """
     Class to get a lazy frame slice.
     Work only with mono segment recording.
+
+    Do not use this classe directly but use `recording.frame_slice(...)`
```
Comment (alejoe91) [inline spikeinterface/core/frameslicerecording.py:10]:
```suggestion
    Do not use this class directly but use `recording.frame_slice(...)`
```

File: spikeinterface/core/unitsselectionsorting.py [hunk 3]
```diff
@@ -8,6 +8,9 @@
 class UnitsSelectionSorting(BaseSorting):
     """
     Class that handles slicin a Sorting object based on a list of unit_ids.
+
+    Do not use this classe directly but use `sorting.select_units(...)`
```
Comment (alejoe91) [inline spikeinterface/core/unitsselectionsorting.py:12]:
```suggestion
    Do not use this class directly but use `sorting.select_units(...)`
```

**Comment by samuelgarcia** [comment]
@alejoe91  : If tests pass could we merge this before your PR ?

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**